data/profiles/default.yaml

```diff
@@ -502,53 +505,3 @@ spec:
       tolerations: []
       podAntiAffinityLabelSelector: []
       podAntiAffinityTermLabelSelector: []
-
-    grafana:
```

Comment:
I think this needs to remain doesn't it?

Reply:
I thought `grafana` is disabled in `default` profile, isn't it? https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/values.yaml#L65